crates/control/src/game_state_filter.rs

```diff
@@ -31,7 +30,6 @@ pub struct CycleContext {
     pub cycle_time: Input<CycleTime, "cycle_time">,
     pub filtered_whistle: Input<FilteredWhistle, "filtered_whistle">,
     pub game_controller_state: RequiredInput<Option<GameControllerState>, "game_controller_state?">,
-    pub sensor_data: Input<SensorData, "sensor_data">,
```

Comment:
I removed this because I initially thought I wanted to run more modules in the behavior cycler, including this one.
And since I didn't want to fake sensor data I just removed this since it was unused anyway.